20_beta_diversity.Rmd

```diff
@@ -135,32 +127,54 @@ p <- p + labs(x = paste("PCoA 1 (", round(100 * rel_eig[[1]], 1), "%", ")", sep
 p
 ```
 
-With additional tools from the ggplot2 package, ordination methods can be 
-compared to find similarities between them or select the most suitable one to
-visualize beta diversity in the light of the research question.
+A few combinations of beta diversity metrics and assay types are conventionally
+used. For instance, Bray-Curtis dissimilarity and Aitchison distance are often applied to the relative abundance and the clr assays, respectively. Besides
+**beta diversity metric** and **assay type**, the **PCoA algorithm** is also a variable that should be considered. Below, we show how the choice of these three
+factors can affect the resulting lower-dimensional data.
 
-```{r plot-mds-nmds-comparison, fig.cap = "Comparison of MDS and NMDS plots based on the Bray-Curtis or euclidean distances on the GlobalPattern dataset."}
+```{r mds-nmds-comparison, warning=FALSE, message=FALSE}
+# Run NMDS on relabundance assay with Bray-Curtis distances
+tse <- runNMDS(tse,
+               FUN = vegan::vegdist,
+               method = "bray",
+               assay.type = "relabundance",
+               name = "NMDS_bray")
+
+# Run MDS on clr assay with Aitchison distances
 tse <- runMDS(tse,
               FUN = vegan::vegdist,
-              name = "MDS_euclidean",
-              method = "euclidean",
-              assay.type = "counts")
+              method = "aitchison",
+              assay.type = "clr",
+              pseudocount = 1,
+              name = "MDS_aitchison")
 
+# Run NMDS on clr assay with Aitchison distances
 tse <- runNMDS(tse,
                FUN = vegan::vegdist,
-               name = "NMDS_BC")
+               method = "aitchison",
+               assay.type = "clr",
+               pseudocount = 1,
```

Comment:
same here

Reply:
Yep. Aitchison applies on counts or relabundances because it does CLR + Euclid automatically. 

Or you can do them separately as well.. but not both at the same time.

Which brings the question back a bit, should we aim to warn users when they do strange combinations? I am not sure, education and good example material could also take us a long way, it is difficult to imagine all unintended use cases.

Reply:
Conventional combinations could be explained in OMA so warning could be avoided?

Reply:
We could mention some of the most typical combinations, yes. These include at least:

- relative abundance transformation + Bray-Curtis dissimilarity index  (for relative microbiota profiling)  
- original counts + Bray-Curtis dissimilarity index (for quantitative microbiota profiling) 
- CLR-transformation + Euclidean distance (= this combination is called Aitchison distance)
- Robust CLR-transformation + Euclidean distance (= this combination is called robust Aitchison distance)

Reply:
Perhaps also worth mentioning: Jaccard index; you can become more similar to something without becoming less similar to something else..